addons/hr/views/hr_employee_views.xml

```diff
@@ -105,9 +105,10 @@
                                             <field name="address_id"
                                                 context="{'show_address': 1}"
                                                 options='{"always_reload": True, "highlight_first_line": True}'/>
+                                            <field name="work_location"/>
                                         </group>
-                                        <group name="managers" string="Responsibles">
-                                            <field name="coach_id"/>
+                                        <group name="managers" string="Approvers">
+                                            <!-- used to contain the coach id field. Keep the group for overrides in other modules -->
```

Comment:
Comments should generally provide information on the current code. You can put: `is overidded in other modules` or something like that